### ISSUE TITLE
Group repeated log entries together

### ISSUE THREAD

**Issue by tigert** [MobiFlight/MobiFlight-Connector#703]
**Is your feature request related to a problem? Please describe.**
Some repeated log entries (ncalc errors, "sim not running") spam the log so its hard to follow what is going on if you want to look for something else there.

![image](https://user-images.githubusercontent.com/2587818/155221328-8b078d69-31c3-4f18-b0ae-0c0a5e8ccd83.png)

**Describe the solution you'd like**
Group the repeated ones into one line and just display the number of repeats, so you can still see if they stop or keep coming.

So if the log entry is the same as previous one (not taking the timestamp into account of course) we could consider them repeats and just show them on that one line:

![image](https://user-images.githubusercontent.com/2587818/155222874-e39a61a7-5dfe-4174-98f6-af3fe88ec68e.png)

**Comment by DocMoebiuz** [comment]
I will rather revisit the log levels and how they are used throughout the code. This looks like spending engineering power on the wrong end of things.

Also try log info level and see if you get the relevant stuff there. 

Actually good question: What is relevant to you out of the logs?

**Comment by neilenns** [comment]
@DocMoebiuz I was gonna say similar... we could just stop writing the execution manager tick event :)

By far the most useful thing for me in the logs is seeing the to/from communication with the Arduino:

- OnPress
- OnRelease
- Encoder actions
- SetOutput
- Startup detection of VID/PIDs and whether board definitions matched

etc.

Basically the stuff that confirms an action did take place and got sent/received. Those could all move to the info log level.

I'm gonna assign this to myself and come back with an inventory of all the log messages and their current levels for you to look at.

**Comment by tigert** [comment]
Yeah, you both hit the nail in the head - the reason I used "debug" is because thats where the input events are.

Lets close this one and continue on #705

**Comment by DocMoebiuz** [comment]
Closing :D
